Userland/Libraries/LibJS/AST.h

```diff
@@ -1781,13 +1797,13 @@ class TaggedTemplateLiteral final : public Expression {
     mutable HashMap<Realm*, Handle<Array>> m_cached_values;
 };
 
-class MemberExpression final : public Expression {
+class MemberExpression : public Expression {
```

Comment:
Noticed that `final` was removed when you showed how padding works on a subclass, but was never added again.

Reply:
Whoops! Thank you so much for spotting that :)